frontend/src/components/utils/NavBar/index.tsx

```diff
@@ -203,19 +252,18 @@ const NavBar = ({ headersData }: Props): ReactElement => {
     </Grid>
   );
 
-  const searchBar = location.pathname !== '/';
-
   const displayDesktop = (): ReactElement => {
     return (
-      <Grid container className={toolbar} alignItems="center">
+      <Grid container className={toolbar} alignItems="center" justifyContent="space-between">
         <Grid item md={3}>
           {homeLogo}
         </Grid>
-        <Grid item md={6} className={searchBar ? search : searchHidden}>
```

Comment:
I believe this line is for adapting new design on homepage. The search bar in navigation page disappear when we're on homepage, otherwise it stays still. Please revert this back then add additional styling so the nav bar appear as expected!